resources/searchview/styles.css

```diff
@@ -81,6 +81,7 @@ button.secondary:hover {
 }
 
 input[type="text"],
+input[type="search"],
```

Comment:
Switch the search input type to `search` to support native clearing of input with a single click vs having to manually delete everything.